src/foremast/utils/gcp_service_account.py

```diff
@@ -0,0 +1,27 @@
+#   Foremast - Pipeline Tooling
+#
+#   Copyright 2020 Redbox Automated Retail, LLC
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Retrieve GCP Credentials"""
+
+from google.oauth2 import service_account
+from ..utils.gcp_environment import GcpEnvironment
+
+
+def get_credentials(env: GcpEnvironment):
```

Comment:
What is the difference between this get_creds call vs the built in one on the class? Do they need to be separate?

Reply:
Left over from a code refactor, this function wasn't used at all.  Fixed!

Reply:
Awesome. Thanks.